### PR TITLE
apache: update to 2.4.53.

### PR DESCRIPTION
**Pull request by ologantr**
Fixes CVE-2022-23943, CVE-2022-22721, CVE-2022-22720,
CVE-2022-22719.

#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture (x86_64-glibc)

Edit: ping @paper42

### CODE REVIEW

File: srcpkgs/apache/template [hunk 1]
```diff
@@ -63,6 +62,10 @@ pre_configure() {
 
 	cat ${FILESDIR}/xbps.layout >> config.layout
 	if [ "$CROSS_BUILD" ]; then
+		# pcre{,2}-config detection doesn't work on cross
+		# since httpd-2.4.53, manually export PCRE_CONFIG
+		# to let configure know where it is
+		export PCRE_CONFIG="${XBPS_CROSS_BASE}/usr/bin/pcre2-config"
```
Comment (paper42) [inline srcpkgs/apache/template:68]: Could you report this issue upstream and watch it so we know when we can drop this workaround?

Comment (ologantr) [reply srcpkgs/apache/template:68]: Turns out it has already been reported: 
https://bz.apache.org/bugzilla/show_bug.cgi?id=66000
I'll follow this issue and wait for it to be fixed.

Comment (paper42) [reply srcpkgs/apache/template:68]: in that case, --with-pcre with the full path might be prettier

Comment (paper42) [reply srcpkgs/apache/template:68]: @ologantr mentioned on IRC that that wouldn't work on cross, so let's use $PCRE_CONFIG for now

**Pull request merged**